framework/src/modules/interoperability/base_interoperability_method.ts

```diff
@@ -120,6 +120,13 @@ export abstract class BaseInteroperabilityMethod<
 		return channel.messageFeeTokenID;
 	}
 
+	public async getMessageFeeTokenIDFromCCM(
+		context: ImmutableMethodContext,
+		ccm: CCMsg,
+	): Promise<Buffer> {
+		return this.getMessageFeeTokenID(context, ccm.receivingChainID);
```

Comment:
maybe to align with the LIP we can use ccm.sendingChainID
https://github.com/LiskHQ/lips/pull/437/files#diff-a3ced5b919eed85370156537064b46119c1e19204e1d348b91f757a0f5ca8d8bR1408-R1414